docs/install/index.md

```diff
@@ -1,5 +1,133 @@
-There are a number of different methods to install and run Coder:
+1. Install Coder
```

Comment:
I think our landing page should include some simple opening header before any bullets, specifically leading with the one-line install cmd. 

---

The easiest way to install Coder is to use our [install script](https://github.com/coder/coder/blob/main/install.sh) for Linux and macOS.

```
$ curl -fsSL https://coder.com/install.sh | sh
```

Reply:
I agree, but I am still looking for a line to start it. We should not recommend isntall_Script as the easiest way. For most people who self-host stuff, docker/docker-compose is easier. It is very subjective and depends on the use case. Installation with the install script is only easy for someone installing Coder on a dedicated VM/host.